app/src/main/java/com/sdp/movemeet/map/MainMapFragment.java

```diff
@@ -111,7 +96,7 @@ public void onMapReady(GoogleMap googleMap) {
         LatLng userLatLng = new LatLng(currentLocation.getLatitude(), currentLocation.getLongitude());
         MarkerOptions markerOptions = new MarkerOptions().position(userLatLng).title("I am here !");
         googleMap.animateCamera(CameraUpdateFactory.newLatLng(userLatLng));
-        googleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(userLatLng, 12.0f));
+        googleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(userLatLng, 15.0f));
```

Comment:
Should this constant be moved to a public static attribute? Maybe we should have a uniform zoom on map fragments.

Reply:
Yes good idea, I've changed it as a static constant called ZOOM_VALUE, used in both the MainMap and the MiniMap